core/sections/clause_8_vector.adoc

```diff
@@ -18,25 +19,44 @@
 
 * color: Color
 
-==== Class Stroke
-===== Class GraphicalUnit
-===== Class StrokeStyling
+==== Class Graphic
 
-==== Class MultiGraphic
-===== Class Marker
+* opacity: float
+* transform: Transform2D
```

Comment:
We need to review _transform_ here. There was discussion that only _transform.translation_ would be supported here.